### PR TITLE
feat: add seo

### PR DESCRIPTION
**Pull request by affo-bien-fonde**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-XXXXX

## ✅ Checklist

I have:

- [ ] Made sure my feature is **working on different browsers** (Chrome, Firefox, Safari).
- [ ] Written **unit tests** for my feature.
- [ ] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change.

## 📸 Screenshots

**Delete** _if no UI change_

| Platforms         | Mockup/Before | After |
| :---------------- | :-----------: | :---: |
| Desktop - Chrome  |               |       |
| Desktop - Firefox |               |       |
| Desktop - Safari  |               |       |
| Phone - Chrome    |               |       |
| Phone - Firefox   |               |       |
| Phone - Safari    |               |       |

### CODE REVIEW

File: public_website/__tests__/fixtures/ressourcesEnseignants.ts [hunk 1]
```diff
@@ -53,6 +52,105 @@ export const RessourcesEnseignatsPageFixtures = {
       publishedAt: '2024-03-19T10:32:38.445Z',
       title:
         'Programmes  pour la  sensibilisation à la musique dans votre région',
+      seo: {
+        id: 1,
+        metaTitle: 'Aide enserignants Resesources',
```
Comment (Lucasbeneston) [inline public_website/__tests__/fixtures/ressourcesEnseignants.ts:57]: Heureusement que ce sont des fixtures :D

File: content_management_system/src/index.ts
Comment (Lucasbeneston) [inline content_management_system/src/index.ts:1]: Cette page sert a quelque chose maintenant ?

File: public_website/src/lib/seo/seo.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        {props.metaData?.structuredData && (
+          <script type="application/ld+json">
+            {JSON.stringify(props.metaData?.structuredData)}
+          </script>
+        )}
+        {props.metaData?.metaViewport && (
+          <meta name="viewport" content={props.metaData?.metaViewport} />
+        )}
+        {props.metaData?.keywords && (
+          <meta name="keywords" content={props.metaData.keywords}></meta>
```
Comment (Lucasbeneston) [inline public_website/src/lib/seo/seo.tsx:34]:
```suggestion
          <meta name="keywords" content={props.metaData.keywords} />
```

File: content_management_system/src/components/shared/meta-social.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "displayName": "metaSocial",
+    "icon": "project-diagram"
+  },
+  "options": {},
+  "attributes": {
+    "socialNetwork": {
+      "type": "enumeration",
+      "enum": [
+        "Facebook",
+        "Twitter"
```
[the remark this answers is not in the document]
Comment (affo-bien-fonde) [reply content_management_system/src/components/shared/meta-social.json]: C'est fait

**Review by Lucasbeneston — APPROVED** [review]

File: content_management_system/src/api/category/content-types/category/schema.json [hunk 4]
```diff
@@ -16,12 +16,6 @@
       "type": "string",
       "required": true,
       "unique": true
-    },
-    "restaurants": {
-      "type": "relation",
-      "relation": "manyToMany",
-      "target": "api::restaurant.restaurant",
-      "inversedBy": "categories"
```
[the remark this answers is not in the document]
Comment (affo-bien-fonde) [reply content_management_system/src/api/category/content-types/category/schema.json:24]: C'est fait

**Review by Lucasbeneston — APPROVED** [review]

**Pull request merged**